.github/workflows/tag-base-images.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - name: Tag upstream images
         if: ${{ startsWith(github.ref, 'refs/heads/release/v') }}
         run: |
-          ./docker/registry/cpp-client-base/build/crane/retag.sh
```

Comment:
Did you mean to remove the cpp?

Reply:
Yes, I Devin request I remove earlier in this same review.  The C++ build is now using the r-client-base image (which we plan to rename `cpp-clients-fat-base` on the py ticking coming PR).